frontend/src/App.tsx

```diff
@@ -0,0 +1,40 @@
+import './App.css'
+import {Link, Outlet, useLocation} from "react-router-dom";
+import {useState} from "react";
+
+function App() {
+    const [user, setUser] = useState("");
```

Comment:
There's a missing feature here: displaying the logged-in user's name after registration.